### PR TITLE
Figma/UI-changes

### PR DESCRIPTION
**Pull request by jatinzop**

### CODE REVIEW

File: src/localization/messages/locally/en-us.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ const en = {
     Link_Expired: "Link expired",
     Reset_password_to_unlock_account:
       "Reset your password or <a_contact_support>Contact Support</a_contact_support> to unlock your account.",
+    "Password_successfully_reset!": "Password successfully reset!",
```
Comment (atulrana007) [inline src/localization/messages/locally/en-us.js:133]: this key already exist Password_successfully_reset       without the ! mark just update that and please check the value for this key as well please alekhya has mentioned new value in the bugs

File: src/localization/messages/locally/en-us.js [hunk 2]
```diff
@@ -130,6 +130,7 @@ const en = {
...
   },
 };
 export default en;
```
Comment (atulrana007) [inline src/localization/messages/locally/en-us.js:136]: please check with rishi as well if all his keys are present in this or not in this

**Comment by atulrana007** [comment]
please resolve this conflict as well

File: src/components/login/passwordless.js [hunk 3]
```diff
@@ -58,7 +72,7 @@ const PasswordLessFlow = (props) => {
               />
               <FormattedMessage id="email">
                 {(msg) => (
-                  <input
+                  <input style={{color: LoginError.isEmailError?"red":""}}
```
Comment (atulrana007) [inline src/components/login/passwordless.js:75]: check line 60 already made this change this is not the right place to please check the component in inspector and then make the change

File: src/providers/CommonDataContext.jsx [hunk 4]
```diff
@@ -23,7 +23,7 @@ const CommonDataProvider = (props) => {
   });
   const [LoginError, setLoginError] = useState({
     email: "",
-    isEmailError: "",
+    isEmailError: false,
```
Comment (atulrana007) [inline src/providers/CommonDataContext.jsx:26]: why are you doing this ? empty string is considered as false

File: src/providers/CommonDataContext.jsx [hunk 5]
```diff
@@ -52,14 +52,15 @@ const CommonDataProvider = (props) => {
     try {
       // `/client/soKVdT2wmzd71LKYoZpv6FJMTg6yQ238.js`
       const res = await axios.get(
-        `https://${props.config.auth0Domain}/client/${props.config.clientID}.js`
+        `/client/soKVdT2wmzd71LKYoZpv6FJMTg6yQ238.js`
```
Comment (atulrana007) [inline src/providers/CommonDataContext.jsx:55]: comment this and uncomment other one

Comment (jatinzop) [reply src/providers/CommonDataContext.jsx:55]: yeah sorry about this, forgot to save before staging

File: src/components/login/passwordless.js [hunk 6]
```diff
@@ -30,7 +42,11 @@ const PasswordLessFlow = (props) => {
               <div
                 className={styles.LoginInputLabel}
                 style={{
-                  color: validateEmail(LoginForm.email) ? "#0CA77D" : "red",
+                  // color: validateEmail(LoginForm.email) ? "#0CA77D" : "red",
```
Comment (atulrana007) [inline src/components/login/passwordless.js:45]: why commented this ?

Comment (jatinzop) [reply src/components/login/passwordless.js:45]: Alekhya said that border is not supposed to turn green according to figma

File: src/providers/CommonDataContext.jsx (hunk 4 above)
Comment (jatinzop) [reply src/providers/CommonDataContext.jsx:26]: Every other key with this kind of naming is given a boolean, just trying consistency.

File: src/components/login/passwordless.js (hunk 3 above)
Comment (jatinzop) [reply src/components/login/passwordless.js:75]: Text colour need to get changed. Line 60 is not subtle for this change.

Comment (atulrana007) [reply src/components/login/passwordless.js:75]: cool

File: src/providers/CommonDataContext.jsx (hunk 4 above)
Comment (atulrana007) [reply src/providers/CommonDataContext.jsx:26]: are we not setting this isEmailError with some string somewhere ?

File: src/components/login/passwordless.js (hunk 6 above)
Comment (atulrana007) [reply src/components/login/passwordless.js:45]: make it to normal grey color then if correct then normal grey color if not valid then red

**Pull request merged**